SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -286,7 +287,7 @@ public static void Main(string[] args)
                 {
                     "ZipUtils",
                     "-add=" + Quote(serverPath),
-                    "-archive=" + Quote(Path.Combine(outputDir, $"UnrealWorker@{assemblyPlatform}.zip"))
+                    "-archive=" + Quote(Path.Combine(outputDir, gameName + $"@{assemblyPlatform}.zip"))
```

Comment:
Should this be `gameName + $"ServerWorker..."`?

Reply:
No, the correct naming is `workername@platform.zip`

Reply:
I think the confusion is because the first argument is used both for game name and worker name and the variable is called `gameName`.

Reply:
so... is `gameName == "UnrealWorker"`?

Reply:
No, it is the game name (normally same as project file name). I think what we should do is kill one of `gameName` or `baseGameName`. The confusion is most clear on the line diff below where we use `gameName` to write the heading but `baseGameName` is passed as an argument to the build script.